src/site/mod.rs

```diff
@@ -140,11 +142,31 @@ impl Site {
                 full_path.parent().unwrap().as_str(),
             )?;
         }
-        if let Some(book_path) = &config.md_book {
-            Self::copy_static(dist, book_path)?;
+        if let Some(book_cfg) = &config.md_book {
```

Comment:
if the preferred key name is `mdbook` we should propagate that throughout the program imo